class LogNoLoggableEntryError(Exception):
    pass


class LogEntryAlreadyInLogError(Exception):
    pass
